pkg/sink/codec/simple/message.go

```diff
@@ -543,16 +550,19 @@ func newDDLMessage(ddl *model.DDLEvent) (*message, error) {
 }
 
 func newDMLMessage(
-	event *model.RowChangedEvent, config *common.Config, onlyHandleKey bool,
+	event *model.RowChangedEvent, config *common.Config,
+	onlyHandleKey bool, claimCheckFileName string,
 ) (*message, error) {
 	m := &message{
-		Version:       defaultVersion,
-		Schema:        event.Table.Schema,
-		Table:         event.Table.Table,
-		CommitTs:      event.CommitTs,
-		BuildTs:       time.Now().UnixMilli(),
-		SchemaVersion: event.TableInfo.UpdateTS,
-		HandleKeyOnly: onlyHandleKey,
+		Version:            defaultVersion,
+		Schema:             event.Table.Schema,
+		Table:              event.Table.Table,
+		TableID:            event.Table.TableID,
```

Comment:
Should use the `ID` from `TableInfo` instead of the `TableID` in `TableName`, because the latter one would be the partition's physical ID.